mimic/test/test_nova.py

```diff
@@ -810,15 +749,86 @@ def test_reboot_server(self):
             treq.json_content(rebooted_server_response))
         self.assertEqual(rebooted_server_response_body['server']['status'], 'ACTIVE')
 
-    def test_change_password(self):
+    def test_rescue(self):
         """
```

Comment:
@lekhajee this is not a test change. It looks like a change in spacing / placement of tests in this file. I  will see about moving them back to original place